src/components/Theme/tokens.scss

```diff
@@ -101,6 +101,7 @@
   --osdk-color-background-button-primary-active: rgba(
     var(--ods-color-background-action-active)
   );
+  --osdk-color-background-button-custom: #d5dae0;
```

Comment:
Since this doesn't look like it's a customisable UI option, I'd avoid using the `osdk` prefix since that's used to denote a supported option. If it is, then it should be documented in README. It probably also needs a more specific naming as right now it's not quite clear what/where this button belongs.

Reply:
Changed to `--color-background-button-custom` and placed some comments to clarify the usage at a very basic level.